layers/+lang/javascript/packages.el

```diff
@@ -260,6 +261,38 @@
           "sR" 'spacemacs/skewer-eval-region-and-focus
           "ss" 'skewer-repl)))))
 
+(defun javascript/init-indium ()
+  (when (eq javascript-repl 'indium)
+    (use-package indium
+      :defer t
+      :init
+      (progn
+        (require 'indium)
+        (add-hook 'js2-mode-hook 'indium-interaction-mode)
+        ;; prefixes
+        (spacemacs/declare-prefix-for-mode 'indium-interaction-mode "mdi" "debugger")
+        (spacemacs/declare-prefix-for-mode 'indium-debugger-mode "mdd" "debugger")
+        ;; keybindings
+        (spacemacs/set-leader-keys-for-minor-mode 'indium-interaction-mode
+          "dil" 'indium-launch
+          "dic" 'indium-connect
+          "die" 'indium-eval-defun
+          "dii" 'indium-inspect-expression
+          "dir" 'indium-repl-switch-from-buffer
+          "did" 'indium-switch-to-debugger
+          "dib" 'indium-toggle-breakpoint
+          "diB" 'indium-remove-all-breakpoints-from-buffer)
+        (spacemacs/set-leader-keys-for-minor-mode 'indium-debugger-mode
```

Comment:
This is the debugger which is ok to be bound here, the question is wouldn't it be nice to also have this minor mode activated when `js2-mode` is started given that the non lsp backends here define no debugging solution?

Of course you need to avoid binding these in case of `lsp` as in this layer `dap` is loaded which is its own debugging server solution.